### DISCUSSION TITLE
Question related to MapUnitToAbbreviation

### DISCUSSION THREAD

**Discussion by alexraoux** [angularsen/UnitsNet#1314] [Q&A]
Hello 
I am using MapUnitToAbbreviation to add some aliases to existing units
For exemple : "um" for micrometers to avoid having to type the greek letter. 

Is there a possibility to make it work for all cultures or should I force the culture myself in all the call of this function and also the ones that parse abbreviation like UnitParser.TryParse(...)

Second question related to the first, is there a possibility to Unmap some aliases ?

**Comment by angularsen** [comment]
1. You need to add and parse for a given culture, or possibly for all supported cultures in UnitsNet if you need it to work for any culture. There is currently no way to enumerate all supported cultures in code, but `en-US` and `ru-RU` are more or less the only ones. You can search the JSON files for all cultures declared. Open to accept pull request if you want to expose all supported cultures as an array or similar, which can be done with the CodeGen project that parses JSON files and produces C# code files.
2. Currently not possible to unmap, but you can add new default abbreviations with `MapUnitToDefaultAbbreviation()`, which is used by `ToString()`. Open to accept pull request if unmapping is still useful.
